components/core/DownloadManager.py

```diff
@@ -441,3 +441,37 @@ def insert_compression_process(comp_id, start_time, completed_time, is_deleted):
 			return e[1]
 		return 0
 	return "db connection error"
+
+def update_minio_indexes(file_name, file_path, gid, completedLength, username, download_id):
+	db = threadpool.connect()
+	if db is not None:
+		cursor = db.cursor()
+		sql = "INSERT into minio(mid, mgid, muser_name, mfile_name, msize, mpath) VALUES(%s, %s, %s, %s, %s, %s);"
+		try:
+			cursor.execute(sql, (download_id, gid, username, file_name, completedLength, file_path))
+			db.commit()
+			cursor.close()
+		except MySQLdb.Error as e:
+			db.rollback()
+			# Shows error thrown up by database
+			print(e)
```

Comment:
I wonder if we should log these errors? 🤔

Reply:
then I guess we should update all related files for logging.

Reply:
OK that is good 👍